frontend/src/metabase/visualizations/echarts/pie/constants.ts

```diff
@@ -1,3 +1,21 @@
+export const DIMENSIONS = {
+  paddingTop: 16,
+  sideLen: 540,
```

Comment:
Should this be calculated based on the dimensions of the chart?

Reply:
For the dynamic chart yes, but for static it's just a fixed value. I'll leave it as a constant since right now we are only doing the static pie chart, but later when we do the dynamic I'll make this dynamic based on chart size.